packages/dds/tree/src/simple-tree/schemaTypes.ts

```diff
@@ -381,15 +382,25 @@ export type ImplicitAllowedTypes = AllowedTypes | TreeNodeSchema;
 export type ImplicitFieldSchema = FieldSchema | ImplicitAllowedTypes;
 
 /**
- * Converts ImplicitFieldSchema to the corresponding tree node's field type.
+ * Converts an `ImplicitFieldSchema` to a property type suitable for reading a field with this that schema.
```

Comment:
Should this type be `@system`?

Reply:
Making this system would be a breaking change (it removes app's ability to use this), so we can't do that, regardless of if we think apps need it.